company_website/view_helpers.py

```diff
@@ -2,11 +2,21 @@
 from typing import Dict
 
 from django.views.generic import TemplateView
+from django.views.generic.base import ContextMixin
 
 from company_website.models import PageSeo
+from project_liberation.settings.production import GOOGLE_ADS_CONVERSION_ID
+from project_liberation.settings.production import GOOGLE_ADS_CONVERSION_TARGET_ADDRESS
 
 
-class CustomTemplateView(TemplateView):
+class GoogleAdsMixin(ContextMixin):
+    extra_context = {
+        "GOOGLE_ADS_CONVERSION_ID": GOOGLE_ADS_CONVERSION_ID,
+        "GOOGLE_ADS_CONVERSION_TARGET_ADDRESS": GOOGLE_ADS_CONVERSION_TARGET_ADDRESS,
```

Comment:
Please tell me, is this acceptable? Because I have this nasty feeling it isn't.
You told me to move those values to the production settings, but these settings are not defined in the base settings file. Should I therefore:
1. Define those settings in the base or development settings and assign them some bogus values?
2. Move them to the base settings?
3. It is completely acceptable to access them here like this and I should shut up. 😛

Reply:
Right, first, we should create this options also in base.py file but there they should be commented out. Second this is to create some mock ids/settings in development.py. It should work then :) Is it nasty solution? I think we haven't better right now and we should keep with this one :)

Reply:
Done.